### PR TITLE
feat(web): add Cerebro web application workspace

### PR DESCRIPTION
**Pull request by jonathanhaaswriter**
## Summary

- add the Cerebro web application under `apps/web`
- wire application lint, test, build, and standalone packaging into root workspace checks
- add architecture tests for the application boundary
- document snapshot-based history migration when historical objects cannot pass the current public-boundary scan

The imported snapshot contains portable application behavior only. It excludes infrastructure automation, deployment configuration, credentials, network topology, and environment-specific operational files.

## Validation

- web lint
- 69 Vitest files and 425 tests
- production Next.js build
- standalone packaging and HTTP smoke checks
- root SDK checks
- `go test ./tools/archtests/... -count=1`
- `make check-structural`
- OSS audit
- public-boundary leak scan
- `git diff --check agent/monorepo-scaffold-main...HEAD`

## Sequence

This PR targets the workspace scaffold in #1936. Once that scaffold is on `main`, this branch can be rebased and merged without any lifecycle-contract dependency.

### CODE REVIEW

File: apps/web/scripts/grc-scale-benchmark.mjs [hunk 1]
```diff
@@ -0,0 +1,1928 @@
...
+          version: "scale-v1",
+          frameworks: [{ id: "soc2", name: "SOC 2", lifecycle: "active", family_count: 5, control_count: 64 }],
+          generated_at: generatedAt,
+        });
+      }
+      return sendJSON(response, stats, normalizedPath, { ok: true, path: normalizedPath, generated_at: generatedAt });
+    } catch (error) {
+      const body = JSON.stringify({ error: error instanceof Error ? error.message : String(error), generated_at: generatedAt });
+      response.writeHead(500, { "content-type": "application/json; charset=utf-8" });
+      response.end(body);
```
[the remark this answers is not in the document]
Comment (jonathanhaaswriter) [reply apps/web/scripts/grc-scale-benchmark.mjs:528]: Fixed in `845e5c6b`: the benchmark response now returns a stable public error code and no exception text.

File: apps/web/scripts/grc-scale-benchmark.mjs [hunk 2]
```diff
@@ -0,0 +1,1928 @@
...
+  while (Date.now() - started < timeoutMs) {
+    try {
+      await action();
+      return;
+    } catch (error) {
+      lastError = error;
+      await sleep(500);
+    }
+  }
+  const logs = currentWebProcess ? await tailLog(`${currentWebProcess.spawnargs?.[0] ?? "web"}`).catch(() => "") : "";
```
[the remark this answers is not in the document]
Comment (jonathanhaaswriter) [reply apps/web/scripts/grc-scale-benchmark.mjs]: Fixed in `845e5c6b`: removed the argument from the zero-argument `tailLog` call.

File: apps/web/src/app/connectors/activation/page.tsx [hunk 3]
```diff
@@ -0,0 +1,895 @@
...
+                  <div className="mt-1 text-[12px] text-[var(--text-muted)]">Run one page for {runtimeID} to verify runtime collection.</div>
+                </div>
+                <button type="button" disabled={syncing} onClick={() => onRunSync(runtimeID)} className="primary-button inline-flex items-center gap-2 px-3 py-2 text-[13px] disabled:opacity-60">
+                  <RefreshCw className={`h-4 w-4 ${syncing ? "animate-spin" : ""}`} />
+                  {syncing ? "Running..." : "Run initial sync"}
+                </button>
+              </div>
+            </div>
+          )}
+          {definition && isDeposit && (
```
[the remark this answers is not in the document]
Comment (jonathanhaaswriter) [reply apps/web/src/app/connectors/activation/page.tsx]: Fixed in `845e5c6b`: removed the redundant `definition` check inside the already narrowed branch.
